website/content/docs/operations/stateful-workloads.mdx

```diff
@@ -205,8 +215,10 @@ choose the storage options that best addresses your Nomad storage requirements.
 | Storage option  | Advantages  | Disadvantages  | Ideal for  |
 |---|---|---|---|
 | CSI volumes  | <ul><li>Wide ecosystem with many providers</li><li>Advanced features such as snapshots, cloning, and resizing</li><li>Dynamic, flexible, and self-service (anyone with the correct ACL policies can create volumes on-demand)</li></ul>    | <ul><li>Some complexity and ongoing maintenance</li><li>Plugin upgrades have to follow the underlying storage provider's API changes/upgrades</li><li>Not all CSI plugins implement all features</li><li>Not all CSI plugins respect the CSI spec and are Nomad compatible</li><li>Node plugins need to run in privileged mode to be able to mount the volumes in allocations</li></ul>  |  <ul><li>Environments where Nomad cluster operators and consumers need to easily add/change storage, and where the storage provider of choice has a CSI plugin that respects the CSI spec</li></ul> |
-| Host volumes backed by local storage  | <ul><li>Readily available</li><li>Fast due to being local</li><li>Doesn't require ongoing maintenance</li></ul>  | <ul><li>Requires coordination between multiple personas to configure and consume (operators running the Nomad clients need to configure them statically in the Nomad client's configuration file)</li><li>Not fault tolerant. In case of hardware failure on a single instance, the data will be lost</li></ul>  | <ul><li> Environments with low persistent storage requirements that could tolerate some failure but prefer not to or have high performance and low latency needs.</li></ul>   |
-| Host volumes backed by networked or clustered storage  | <ul><li>Readily available</li><li>Require no ongoing maintenance on the Nomad side (but might on the storage provider) </li></ul>  | <ul><li>Require coordination between multiple personas to configure and consume (storage admins need to provision volumes, operators running the Nomad clients need to configure them statically in the Nomad client's configuration file)</li><li>The underlying networked storage and its limitations are decoupled from the consumer, but need to be understood. For example, is concurrent access possible</li></ul>  | <ul><li>Environments with low amounts or low frequency of change of storage that have an existing storage provider that can be consumed via NFS/CIFS.</li></ul>  |
+| Dynamic host volumes backed by local storage  | <ul><li>Readily available</li><li>Fast due to being local</li><li>Doesn't require ongoing maintenance</li></ul>  | <ul><li>Not fault tolerant. In case of hardware failure on a single instance, the data will be lost unless the application can restore the data</li></ul>  | <ul><li>Environments with high performance and low latency persistent storage requirements where the application is designed to tolerate node failures.</li></ul>   |
```

Comment:
this table is quite hard to read now that it takes up ~3 pages of scroll (on my monitor).

not sure how readily doable this is with our `mdx`, but I think it could be better displayed with labels like "ecosystem", "customizable", "speed", "complexity", "fault tolerant", etc, with checkmarks or numbers 1-3 or something.

I guess this is sometimes called a [comparison table](https://www.nngroup.com/articles/comparison-tables/)?

I don't have a good answer though for how the extra informative language could be elaborated on. maybe this long, wordy table stays as-is, but with a high-level comparison table for a quick glance?

![comparison table](https://media.nngroup.com/media/editor/2024/02/02/comparison-table-taylor_comaprison-table1.png)

Reply:
> this table is quite hard to read now that it takes up ~3 pages of scroll (on my monitor).

Yeah I was just keeping the existing structure here, which came from the SME folks. I feel like checkbox categories are going to squash a lot of nuance. But maybe that's ok if the supporting text has that nuance? There's some dubious items here like "requires no maintenance".

Reply:
I agree. I'd rather see a comparison table with checkmarks. I'll create a ticket to look at refactoring this later.